README.md

```diff
@@ -172,82 +100,92 @@ In Device Simulator Express, you can use keyboard to interact with the device:
   - 25 LEDs
   - Light sensor
   - Motion sensors
-  - Acceleration detection
+  - Acceleration detection including gesture detection
   - Temperature sensor
 
 ### Useful Links
 - Tutorials and Example Code for BBC micro:bit:
   - [MicroPython documentation](https://microbit-micropython.readthedocs.io/en/latest/)
   - [BBC micro:bit examples on the official micro:bit website](https://microbit.org/projects/make-it-code-it/?filters=python)
 
-### How to use
+### Keybindings
+- Push Button `A for A, B for B, C for A & B`
+- Refresh the simulator: `SHIFT + R`
+- Run the simulator: `SHIFT + F`
 
-#### I. Take a look at the "Device Simulator Express: Getting Started" Command.
-1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
+## How to use
+
+To use Device Simulator Express, install the extension from the marketplace and reload VS Code.
+
+### I. Take a look at the "Device Simulator Express: Getting Started" Command.
+1. Type in `"Device Simulator Express: Getting Started"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  

```

Comment:
These didn't format well either (maybe the image messed it up?). You can follow the pattern of the steps below and add the image before any steps (although I personally prefer the image afterwards).

Reply:
I put the image after all of them and it seemed to have fixed the indent issue

Reply:
Cool!